pkg/cmd/server/api/helpers.go

```diff
@@ -0,0 +1,71 @@
+package api
+
+import (
+	"errors"
+	"io/ioutil"
+
+	kclient "github.com/GoogleCloudPlatform/kubernetes/pkg/client"
+)
+
+func GetData(i FileOrData) ([]byte, error) {
+	if len(i.Data) > 0 {
+		return i.Data, nil
+	}
+
+	if len(i.File) > 0 {
+		return ioutil.ReadFile(i.File)
+	}
+
+	return nil, errors.New("no data source specified")
+}
+
+func GetClientConfig(serverURL string, caFile string, clientCertInfo CertInfo) (*kclient.Config, error) {
+	caData, err := ioutil.ReadFile(caFile)
```

Comment:
Why isn't this part of something upstream?  Why are we doing very similar things?